verafelix/reprograma.py

```diff
@@ -0,0 +1,202 @@
+dataset = {
+    ("Ana", "Silva"): {
+        "Turma": "Turma A",
+        "Notas": [7.5, 8.0, 9.0],
+        "Presença": [True, True, False, True, True],
+        "Participação": 8.5
+    },
+    ("Mariana", "Santos"): {
+        "Turma": "Turma B",
+        "Notas": [6.0, 7.5, 8.5],
+        "Presença": [True, True, True, False, True],
+        "Participação": 7.2
+    },
+    ("Carla", "Oliveira"): {
+        "Turma": "Turma A",
+        "Notas": [8.0, 7.5, 8.5],
+        "Presença": [True, True, True, True, True],
+        "Participação": 8.2
+    },
+    ("Juliana", "Ferreira"): {
+        "Turma": "Turma C",
+        "Notas": [9.0, 8.5, 7.0],
+        "Presença": [True, True, True, True, True],
+        "Participação": 8.7
+    },
+    ("Patrícia", "Souza"): {
+        "Turma": "Turma B",
+        "Notas": [7.0, 7.0, 7.5],
+        "Presença": [True, False, True, True, True],
+        "Participação": 7.2
+    },
+    ("Aline", "Martins"): {
+        "Turma": "Turma A",
+        "Notas": [8.5, 8.0, 9.0],
+        "Presença": [True, True, True, True, True],
+        "Participação": 8.5
+    },
+    ("Fernanda", "Costa"): {
+        "Turma": "Turma C",
+        "Notas": [6.5, 7.0, 8.0],
+        "Presença": [True, True, True, False, True],
+        "Participação": 7.2
+    },
+    ("Camila", "Pereira"): {
+        "Turma": "Turma B",
+        "Notas": [7.5, 8.0, 8.5],
+        "Presença": [True, True, True, True, True],
+        "Participação": 8.0
+    },
+    ("Luana", "Rodrigues"): {
+        "Turma": "Turma A",
+        "Notas": [9.0, 9.0, 8.5],
+        "Presença": [True, True, True, True, True],
+        "Participação": 8.8
+    },
+    ("Beatriz", "Lima"): {
+        "Turma": "Turma C",
+        "Notas": [8.0, 7.5, 7.0],
+        "Presença": [True, True, True, False, True],
+        "Participação": 7.5
+    }
+}
```

Comment:
Manter essa estrutura selecionada ou deixar apenas o import possui o mesmo efeito no código. Não era necessário trazer os dados na mão, apenas deixar do modo como estava:

**Sugestão:**
```suggestion
from dataset_alunas import dataset
```